pkg/loop/service.go

```diff
@@ -30,7 +30,7 @@ type grpcPlugin interface {
 
 // pluginService is a [types.Service] wrapper that maintains an internal [types.Service] created from a [grpcPlugin]
 // client instance by launching and re-launching as necessary.
-type pluginService[P grpcPlugin, S services.Service] struct {
+type PluginService[P grpcPlugin, S services.Service] struct {
```

Comment:
Why does this need to be exported?

Reply:
It needs to be exported so it can be used by `pkg/loop/reporting_plugins`